tests/models/user.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.utils import timezone
+
+import pytest
+
+import accounts
+from pootle_statistics.models import (SubmissionFields,
+                                      SubmissionTypes, Submission)
+from pootle_store.models import SuggestionStates
+
+from .store import _update_from_upload_file
+
+
+def _create_submission_and_suggestion(store, user,
+                                      filename=None, suggestion=None):
+    filename = filename or "tests/data/po/tutorial/en/tutorial_update.po"
+
+    # Update store as user
+    _update_from_upload_file(store, filename, user=user)
+
+    # Add a suggestion
+    unit = store.units.first()
+    unit.add_suggestion(suggestion or 'SUGGESTION', user=user)
```

Comment:
Instead of using `suggestion or 'SUGGESTION'`, setting the default parameter to the function as `suggestion='SUGGESTION'` is straightforward and natural. Same for `filename` above.